### ISSUE TITLE
Translation TEMPLATES folder information

### ISSUE THREAD

**Issue by Turbovix** [tananaev/traccar#3352]
Anton, it is possible that the informations in the TEMPLATES folder is also translated via TRANSIFEX, I know that it is possible to do the editing manually, and that is how I do it, however whenever there is an update I have to do it.

**Comment by tananaev** [comment]
I agree that we should implement this at some point. I'm sure it's possible to translate simple text template through transifex, but I'm not sure about Excel reports.

**Comment by Turbovix** [comment]
I understand, however, email messages and sms are useful, since they are used more frequently.

Thank you!

**Comment by tananaev** [comment]
I checked what Transifex provides and I believe it will probably be easier to localize templates separately. Just files as they are. All we need to do is to provide localization support to the whole template system.

**Comment by Turbovix** [comment]
I understand, so let's aguadar the availability in the Transifex to do the translations!

Thank you!

**Comment by tananaev** [comment]
I don't know what "aguadar" means.

**Comment by Turbovix** [comment]
Sorry, I meant "wait"!

**Comment by tananaev** [comment]
I don't think it will ever be available there. We can just do translations directly in Traccar repository, but first we need to implement support for it.

**Comment by Turbovix** [comment]
Okay, so when it's available to do the translations, it'll be great!

**Comment by macano13** [comment]
Ok, can we do this, email, sms template translation?
